lib/api/v3/utilities/path_helper.rb

```diff
@@ -215,6 +215,10 @@ def self.news(id)
           index :placeholder_user
           show :placeholder_user
 
+          def self.show_placeholder(user_id)
+            placeholder_user_path(user_id)
+          end
```

Comment:
Why that indirection, the show page is no v3 path after all. IMO you could just use the `placeholder_user_path` in the representer.